lib.php

```diff
@@ -1185,15 +1215,13 @@ function questionnaire_reset_userdata($data) {
  * Obtains the automatic completion state for this questionnaire based on the condition
  * in questionnaire settings.
  *
- * @param stdClass $course Course
- * @param stdClass $cm Course-module
+ * @param object $cm Course-module
  * @param int $userid User ID
  * @param bool $type Type of comparison (or/and; can be used as return value if no conditions)
  * @return bool True if completed, false if not, $type if conditions not set.
  *
- * $course is unused, but API requires it. Suppress PHPMD warning.
  */
-function questionnaire_get_completion_state($course, $cm, $userid, $type) {
+function questionnaire_get_completion_state_lib($cm, $userid, $type) {
```

Comment:
Why has this been renamed? Since the callback function is now replaced with the "completion\custom_completion" class, shouldn't this be removed? Does renaming it allow it to be used somewhere else?